.github/workflows/cibuildwheel.yml

```diff
@@ -73,7 +86,8 @@ jobs:
           python -m cibuildwheel --output-dir dist
         env:
           CIBW_BUILD: "pp3?-*"
-          CIBW_BEFORE_BUILD: pip install certifi numpy==1.16
+          CIBW_BEFORE_BUILD: pip install certifi oldest-supported-numpy
+          CIBW_ARCHS_LINUX: ${{ matrix.cibw_archs }}
```

Comment:
```suggestion
          CIBW_ARCHS: ${{ matrix.cibw_archs }}
```

Same.

Reply:
Gah! I changed the variable name but forgot the env variable.